src/source/DataModelV2/LevelInstance.cpp

```diff
@@ -43,11 +115,28 @@ std::vector<PROPGRIDITEM> LevelInstance::getProperties()
 		"The amount of time in seconds the player has to complete this level.\r\n\r\nPut 0 if time is limitless.",
 		(LPARAM)timerTxt,
 		PIT_EDIT));
+
 	properties.push_back(createPGI("Gameplay",
 		"InitialScoreValue",
 		"The amount of points the player starts with.",
 		(LPARAM)scoreTxt,
 		PIT_EDIT));
+
+		properties.push_back(createPGI("Gameplay",
+		"TimerUpAction",
+		"Some temporary string here",
+		(LPARAM)strActionType(TimerUpAction),
```

Comment:
Oh I see

Reply:
Can i merge this branch to develop branch?